.env.example

```diff
@@ -1,5 +1,10 @@
-API_CLIENT_ID=<Client ID>
-API_SECRET=<Access key>
-#API_PRODUCT_TYPE=employment
-#API_PRODUCT_TYPE=income
-#API_PRODUCT_TYPE=admin
\ No newline at end of file
+# please set your <Client ID>
+API_CLIENT_ID=
+
+# please set your <Access key>
+API_SECRET=
+
+# you can choose a different product type, just uncomment the line
+API_PRODUCT_TYPE=employment
```

Comment:
could we add `dds` and `pll` as well?

Reply:
ok. I'll check that its available for each languages

Reply:
@chetvil done ;)